source/auth_n.rst

```diff
@@ -4962,7 +4914,7 @@ Challenging During Factor Creation
 
 .. only:: rest
 
-  To send a challenge at the same time as you create the phone Factor, you need to POST to the Account's ``/factors`` endpoint with the additional ``?challenge=true`` parameter included. Then you must also add the ``challenge`` into the body of the JSON.
+  To send a challenge at the same time as you create the SMS Factor, you need to POST to the Account's ``/factors`` endpoint with the additional ``?challenge=true`` parameter included. Then you must also add the ``challenge`` into the body of the JSON.
```

Comment:
This should not appear in Java, this is rest. Not sure this is an unintended change or not.